### ISSUE TITLE
MFA isssue

### ISSUE THREAD

**Issue by jyjohnson** [aws-azure-login/aws-azure-login#9]
We have the text verification code enabled (when logging in off corporate network).

I think this code is not capturing the verification box correctly:
       const tfaResult = content("#tfa_results_container>div").filter(function () {
            return content(this).css('display') === 'block';
        });

This is how the text box html is rendered (in IE):

```
<div class="login_textfield tfa_textfield no_display" id="tfa_code_container" style="display: block;">
    <span class="tfa_input_field tfa_textfield">
        <label class="no_display" aria-hidden="true" for="tfa_code_inputtext">
            Verification code</label>
        <div class="input_border high_contrast_border">
            <span class="placeholder normaltext" id="tfa_code_inputtext_placeholder_span" style="display: block; visibility: visible; position: absolute;">Enter verification code</span><span class="placeholder normaltext" id="tfa_code_inputtext_placeholder_span" style="display: block; visibility: visible; position: absolute;"> </span>
<input name="code" class="login_textfield textfield required tfa_field normaltext" id="tfa_code_inputtext" style="background: none;" spellcheck="false" aria-label="Verification code" type="text" placeholder=" " value="" autocomplete="off">
        </div>
    </span>
</div>
```

Any ideas?

Note the Azure login page is showing a prompt for a "new" interface... Assuming the nodejs script is staying with the default option.

Thanks!

**Comment by Fodoj** [comment]
We are having the same issue here.

**Comment by rmsrohan** [comment]
Same problem.

**Comment by dtjohnson** [comment]
Microsoft must have changed something. It's still working properly for the app verification; it's just the text message verification that seems broken. For some reason the two text message divs are not showing up as visible in PhantomJS. I'll need to dig in some more...

**Comment by Fodoj** [comment]
I actually have app verification, not text

**Comment by dtjohnson** [comment]
It seems that there is some delay between when the page loads and when the appropriate message is made visible. I'll put some fix in to check on a time interval. Hopefully I'll get to this tonight.

**Comment by jyjohnson** [comment]
ok this solved it for me (works if the verification code box is shown or hidden):

line 154: `const tfaResult = content("#tfa_title_text");`
line 164: `if (tfaMessage && tfaMessage.toLowerCase().indexOf("verify your account") >= 0) {`

The 'title text' is immediately displayed but as author pointed out, the verification box has slight delay.

**Comment by dtjohnson** [comment]
K. I just published v0.6.2 with a time delay loop that seems to resolve this.

**Comment by Fodoj** [comment]
I confirm that v0.6.2. works for me.
